pallets/asset/src/lib.rs

```diff
@@ -365,17 +365,24 @@ decl_module! {
             use polymesh_primitives::migrate::{migrate_double_map, Migrate};
             use polymesh_primitives::document::DocumentOld;
             use sp_std::collections::btree_map::BTreeMap;
-            let mut id_map = BTreeMap::<_, u32>::new();
-            migrate_double_map::<_, _, Blake2_128Concat, _, _, _, _, _>(
+
+            let spec_version = frame_system::LastRuntimeUpgrade::get().map_or(0, |upgrade| upgrade.spec_version.0);
```

Comment:
Not sure that we need this logic. We would just rip out the relevant runtime upgrade stuff once Alcyone has been moved to 2.3 (which includes this upgrade logic)?

Reply:
New logic based on local `StorageVersion`.